SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -243,6 +174,142 @@ private static int CreateDeployment(string[] args)
+            numSimulatedPlayersFlag.Add("name", "target_num_simulated_players");
+            numSimulatedPlayersFlag.Add("value", $"{simNumPlayers}");
+
+            var simWorkerConfigJson = File.ReadAllText(simDeploymentJsonPath);
+            dynamic simWorkerConfig = JObject.Parse(simWorkerConfigJson);
+
+            for (var i = 0; i < simWorkerConfig.workers.Count; ++i)
+            {
+                if (simWorkerConfig.workers[i].worker_type == CoordinatorWorkerName)
+                {
+                    simWorkerConfig.workers[i].flags.Add(devAuthTokenIdFlag);
+                    simWorkerConfig.workers[i].flags.Add(targetDeploymentFlag);
+                    simWorkerConfig.workers[i].flags.Add(numSimulatedPlayersFlag);
+                }
+            }
+
+            // Specify the number of managed coordinator workers to start by editing
+            // the load balancing options in the launch config. It creates a rectangular
+            // launch config of N cols X 1 row, N being the number of coordinators
+            // to create.
+            // This assumes the launch config contains a rectangular load balancing
+            // layer configuration already for the coordinator worker.
+            var lbLayerConfigurations = simWorkerConfig.load_balancing.layer_configurations;
+            for (var i = 0; i < lbLayerConfigurations.Count; ++i)
+            {
+                if (lbLayerConfigurations[i].layer == CoordinatorWorkerName)
+                {
+                    var rectangleGrid = lbLayerConfigurations[i].rectangle_grid;
+                    rectangleGrid.cols = simNumPlayers;
+                    rectangleGrid.rows = 1;
+                }
+            }
+
+            simWorkerConfigJson = simWorkerConfig.ToString();
+
+            // Create simulated player deployment.
+            var simDeploymentConfig = new Deployment
+            {
+                AssemblyId = assemblyName,
+                LaunchConfig = new LaunchConfig
+                {
+                    ConfigJson = simWorkerConfigJson
+                },
+                Name = simDeploymentName,
+                ProjectName = projectName,
+                RegionCode = regionCode
+                // No snapshot included for the simulated player deployment
+            };
+
+            simDeploymentConfig.Tag.Add(DEPLOYMENT_LAUNCHED_BY_LAUNCHER_TAG);
+            simDeploymentConfig.Tag.Add(SIM_PLAYER_DEPLOYMENT_TAG);
+
+            Console.WriteLine(
+                $"Creating the simulated player deployment {simDeploymentName} in project {projectName} with {simNumPlayers} simulated players. Link: https://console.improbable.io/projects/{projectName}/deployments/{simDeploymentName}/overview");
+
+            var simDeploymentCreateOp = deploymentServiceClient.CreateDeployment(new CreateDeploymentRequest
+            {
+                Deployment = simDeploymentConfig
+            });
+
+            return simDeploymentCreateOp;
+        }
+
+
```

Comment:
```suggestion
```

Reply:
This branch has been rebased on top of @WVerlaek's branch, which is why these files appear now 😞. The merging of this branch is postponed until the other branches make it to the master.